src/components/filterBar/view/filterBarView.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { View, Text, TouchableOpacity } from 'react-native';
 import { Icon } from '../../icon';
+import { ButtonGroup } from 'react-native-elements';
```

Comment:
Can we change with our ui elements, we are just using button group but importing all react-native-elements features. We have tags button group, i think we can use that, maybe you cannot find in buttons folder

Reply:
yes, just now pushed comment with use of Tag component, removed rn-elements